ref/general/auth-filter.adoc

```diff
@@ -6,20 +6,20 @@
 // Contributors:
 //     IBM Corporation
 //
-:page-description: You can use authentication filters to specify how Open Liberty processes authentication requests from different users and sources.
+:page-description: You can use authentication filters to specify which authentication mechanism Open Liberty uses to processes different HTTP servlet requests.
 :seo-title: Using authentication filters
-:seo-description: You can use authentication filters to specify how Open Liberty processes authentication requests from different users and sources.
+:seo-description: You can use authentication filters to specify which authentication mechanism Open Liberty uses to processes different HTTP servlet requests.
 :page-layout: general-reference
 :page-type: general
-= Using authentication filters
+= Authentication filters
 
-You can use authentication filters to specify how Open Liberty processes authentication requests from different users and sources.
+You can use authentication filters to specify which authentication mechanism Open Liberty uses to processes different HTTP servlet requests.
```

Comment:
"Open Liberty uses to process" (3 times, because I think this is in the page/seo descriptions)